input/postgres/relations.go

```diff
@@ -32,6 +38,9 @@ const relationsSQL string = `
 				%s,
 				COALESCE((SELECT inhparent FROM pg_inherits WHERE inhrelid = c.oid ORDER BY inhseqno LIMIT 1), 0) AS parent_relid,
 				%s,
+				%s,
+				%s,
+				%s,
```

Comment:
Should we keep all the fields together to avoid the interpolation proliferation? I saw that's done in some places but not others...

Reply:
I think this is okay for now, but open to adjusting as well if you think there is a better way to structure it.